From: Jan Engelhardt <ej@inai.de>
Date: 2024-11-02 13:08:11.371010482 +0100
Upstream: already fixed in git snapshots

mifare_desfire_crypto.c:142:9: error: implicit declaration of function 'abort' [-Wimplicit-function-declaration]

---
 libfreefare/mifare_desfire_crypto.c |    1 +
 1 file changed, 1 insertion(+)

Index: libfreefare-libfreefare-0.4.0/libfreefare/mifare_desfire_crypto.c
===================================================================
--- libfreefare-libfreefare-0.4.0.orig/libfreefare/mifare_desfire_crypto.c
+++ libfreefare-libfreefare-0.4.0/libfreefare/mifare_desfire_crypto.c
@@ -57,6 +57,7 @@
 #include <openssl/des.h>
 
 #include <err.h>
+#include <stdlib.h>
 #include <string.h>
 #include <strings.h>
 
